blocks/init/src/Blocks/custom/tabs/components/tabs-panel.js

```diff
@@ -0,0 +1,39 @@
+import { selector } from '@eightshift/frontend-libs/scripts/helpers';
+import React from 'react';
+import manifest from './../manifest.json';
+
+export const TabsPanel = (props) => {
+
+    const { childBlocks, activeTab, setAttributes } = props;
+    const tabItems = childBlocks.map((block) => 
+        {
+            let classes = selector(manifest.tabPanelItemElement, manifest.blockClass, manifest.tabPanelItemElement);
+            if(activeTab === block.tabId) classes += ` ${manifest.tabPanelItemActiveClass}`;
```

Comment:
We try not to use inline conditionals like this (unless they're shorthand when you're assigning a value).

So either:

```js
if (activeTab === block.tabId) {
  classes += ` ${manifest.tabPanelItemActiveClass}`;
}
```

or 

```js
classes += activeTab === block.tabId ? ` ${manifest.tabPanelItemActiveClass}` : '';
```